benchmark/src/Encoding.cpp

```diff
@@ -32,21 +32,17 @@
 #define _USE_MATH_DEFINES
 #include "benchmark/benchmark.h"
 
-bool runOnlyOnce = true;
+bool runOnlyOnce = true; // TODO (dsuponit): do we need runOnlyOnce???
```

Comment:
Well, do we?

Reply:
I added comments as I didn't want my finding to be lost. All those things are to be corrected/changed later.